app/client/templates/layouts/master_layout/master_layout.html

```diff
@@ -28,7 +57,4 @@
       </div>
     </div>
   </div>
-  <div>
-    {{> yield}}
-  </div>
-</template>
+</template>
```

Comment:
Oh, ESLint didn't catch that I suppose because they are HTMLs. I will review that.